### PR TITLE
ubuntu/bootstrap: use debian repository api to get latest version

### PR DESCRIPTION
**Pull request by yuwash**
Issue #3145 (reading the currently utf-16-encoded html seems difficult)

### CODE REVIEW

**Comment by yuwash** [comment]
I signed it!11.03.2017, 00:42, "googlebot" <notifications@github.com>:Thanks for your pull request.  It looks like this may be your first contribution to a Google open source project. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
<g-emoji alias="memo" fallback-src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f4dd.png" ios-version="6.0">📝 Please visit https://cla.developers.google.com/ to sign.
Once you've signed, please reply here (e.g. I signed it!) and we'll verify.  Thanks.


If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your email is set on your git commits.
If you signed the CLA as a corporation, please let us know the company's name.

File: installer/ubuntu/bootstrap [hunk 1]
```diff
@@ -15,9 +15,10 @@ if true || ! wget -O- --no-verbose --timeout=60 -t2 "$d"  \
         | tar -C "$tmp" --strip-components=1 -zx 2>/dev/null; then
     echo 'Download from Debian gitweb failed. Trying latest release...' 1>&2
     d='http://httpredir.debian.org/debian/pool/main/d/debootstrap/'
-    f="`wget -O- --no-verbose --timeout=60 -t2 "$d" \
-            | sed -ne 's ^.*\(debootstrap_[0-9.]*.tar.gz\).*$ \1 p' \
-            | tail -n 1`"
+    d_api='https://sources.debian.net/api/src/debootstrap/'
+    f="`wget -O- --no-verbose --timeout=60 -t2 "$d_api" \
+            | sed -ne 's ^.*\"version\":\ \"\([0-9.]*\)\".*$ debootstrap_\1.tar.gz p' \
```
Comment (dnschneid) [inline installer/ubuntu/bootstrap]: Is `sort -n` going to give the correct order?  Don't you need `sort -V`?

Comment (drinkcat) [inline installer/ubuntu/bootstrap]: Right, `sort -V`

Comment (dnschneid) [inline installer/ubuntu/bootstrap]: sort -V | tail -n 1

Comment (yuwash) [inline installer/ubuntu/bootstrap]: Yes, your suggestions seem to work; and good not to rely too much on the current api representation; fixed in the new commit

**Comment by drinkcat** [comment]
LGTM, thanks!

**Pull request merged**